fortran/test/fetch_content_integration/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 
 include(FetchContent)
 
-set(MUSICA_GIT_TAG "main" CACHE STRING "Git tag for the musica_fortran repository")
+set(MUSICA_GIT_TAG "801807d" CACHE STRING "Git tag for the musica_fortran repository")
```

Comment:
```suggestion
set(MUSICA_GIT_TAG "main" CACHE STRING "Git tag for the musica_fortran repository")
```

Reply:
I changed it back to `main`. I guess we don't worry about failing the test for fetch content if it always sets to `main`?

Reply:
The CI uses the current branch name for the tests when you make a PR